### PR TITLE
Add documentation and tests

### PR DESCRIPTION
**Pull request by Nour-Cheour10**

### CODE REVIEW

**Comment by Nour-Cheour10** [comment]
please update snapshots

File: lite_build_output/jupyter_lite_config.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "LiteBuildConfig": {
+    "contents": ["/home/nourcheour/Ipy-openlayers/docs/content"],
+    "output_dir": "/home/nourcheour/Ipy-openlayers/lite_build_output/dist"
```
Comment (martinRenou) [inline lite_build_output/jupyter_lite_config.json]: You have paths that are specific to your machine here

**Review by martinRenou — COMMENTED** [review]
Thanks!

Overall I think the PR looks good!

We'll need to move the content of `lite_build_output` (part of it at least) into the `docs` directory and make use of jupyterlite-sphinx instead of doing a manual build of jupyterlite, so that the jupyterlite build is done during the docs build automatically (need for readthedocs).

**Comment by martinRenou** [comment]
Triggering CI

**Comment by martinRenou** [comment]
please update snapshots

**Comment by martinRenou** [comment]
Triggering CI

**Pull request merged**